### PR TITLE
Add how to setup LXD access to FAQ (fixes #1281)

### LINKED ISSUE #1281: How to make LXD server public?
I couldn't find documentation how to make LXD server accessible from local network? Is there any command a local client can do?

`lxc config get core.https_address` doesn't work.

### PR DESCRIPTION
**Pull request by techtonik**
Signed-off-by: anatoly techtonik <techtonik@gmail.com>

### CODE REVIEW

**Comment by stgraber** [comment]
This pull request didn't trigger Jenkins as its author isn't in the whitelist.

An organization member must perform one of the following:

 - To have this branch tested by Jenkins, use the "ok to test" command.
 - To have a one time test done, use the "test this please" command.

Those commands are simple Github comments of the format: "jenkins: COMMAND"

File: README.md [hunk 1]
```diff
@@ -118,8 +118,42 @@ find and subscribe to those at: https://lists.linuxcontainers.org
 If you prefer live discussions, some of us also hang out in
 [#lxcontainers](http://webchat.freenode.net/?channels=#lxcontainers) on irc.freenode.net.
 
+
 ## FAQ
 
+#### How to enable LXD server for remote access?
+
+By default LXD server is not accessible from network, because it listens on
```
Comment (stgraber) [inline README.md]: from the network

File: README.md [hunk 2]
```diff
@@ -118,8 +118,42 @@ find and subscribe to those at: https://lists.linuxcontainers.org
...
+local socket. You can make LXD available from the network by specifying
```
Comment (stgraber) [inline README.md]: it only listens on a local unix socket.

File: README.md [hunk 3]
```diff
@@ -118,8 +118,42 @@ find and subscribe to those at: https://lists.linuxcontainers.org
...
+additional interface to listen to. This is done with `core.https_address`
```
Comment (stgraber) [inline README.md]: This is done with the

File: README.md [hunk 4]
```diff
@@ -118,8 +118,42 @@ find and subscribe to those at: https://lists.linuxcontainers.org
...
+config variable.
+
+To see current server configuration, issue:
```
Comment (stgraber) [inline README.md]: the current server configuration

File: README.md [hunk 5]
```diff
@@ -118,8 +118,42 @@ find and subscribe to those at: https://lists.linuxcontainers.org
...
+
+    lxc config show
+
+To set address to listen to, find out what addresses are available and use
```
Comment (stgraber) [inline README.md]: To set the address

File: README.md [hunk 6]
```diff
@@ -118,8 +118,42 @@ find and subscribe to those at: https://lists.linuxcontainers.org
...
+`config set` command on server:
```
Comment (stgraber) [inline README.md]: on the server

File: README.md [hunk 7]
```diff
@@ -118,8 +118,42 @@ find and subscribe to those at: https://lists.linuxcontainers.org
...
+
+By default, LXD has no password for security reasons, so you can't do a remote
+add this way. In order to set a password, do:
+
+    lxc config set core.trust_password SECRET
+
+on the host LXD is running on. This will set the remote password that you can
+then use to do `lxc remote add`.
+
+You can also access server without setting password by copying client
```
Comment (stgraber) [inline README.md]: access the server
without setting a password
by copying the client

File: README.md [hunk 8]
```diff
@@ -118,8 +118,42 @@ find and subscribe to those at: https://lists.linuxcontainers.org
...
+certificate from `.config/lxc/client.crt` to server and adding it with:
```
Comment (stgraber) [inline README.md]: to the server

**Comment by stgraber** [comment]
Just a few grammar tweaks :)

**Comment by techtonik** [comment]
I'll make edits tomorrow. Is there any sane tutorial about those articles? I need to read upon rules how to analyze the surrounding context to insert/remove them correctly. Too bad English is not a programming language to understand those right. Is it something of a type declaration or a context hint?

**Comment by techtonik** [comment]
@stgraber Done.

**Pull request merged**